src/output/icons.rs

```diff
@@ -685,8 +717,10 @@ const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
     "windows"        => Icons::OS_WINDOWS,       // 
     "wma"            => Icons::AUDIO,            // 
     "wmv"            => Icons::VIDEO,            // 
+    "webloc"         => '\u{f0ac}',              // 
```

Comment:
Seems like \u{f1fa} could be a better choice.  Should make a constant if using multiple times.